chartfx-chart/src/main/java/io/fair_acc/chartfx/plugins/Zoomer.java

```diff
@@ -794,11 +794,11 @@ private Map<Axis, ZoomState> getZoomDataWindows() {
             double dataMin;
             double dataMax;
             if (axis.getSide().isVertical()) {
-                dataMin = axis.getValueForDisplay(minPlotCoordinate.getY());
-                dataMax = axis.getValueForDisplay(maxPlotCoordinate.getY());
+                dataMin = axis.getValueForDisplay(Math.min(axis.getHeight(), minPlotCoordinate.getY()));
+                dataMax = axis.getValueForDisplay(Math.min(axis.getHeight(), maxPlotCoordinate.getY()));
```

Comment:
```suggestion
                dataMax = axis.getValueForDisplay(Math.max(0, Math.min(axis.getHeight(), maxPlotCoordinate.getY())));
```

Is this sufficient or should it also clamp to `>0`? That would probably the rarer case, but I don't see any fundamental reason for only limiting one direction.

Reply:
I thought about this case too and would say that it either never occurs or is rare.
I committed you suggested change :-).